### DISCUSSION TITLE
localtileserver

### DISCUSSION THREAD

**Discussion by Mahnaz-Meem** [giswqs/geohey#21] [Q&A]
I have some issues with localtileserver. I have installed it but it still gives me the following error when I try to run it:
![image](https://github.com/giswqs/geohey/assets/108950657/8201d499-89fc-41aa-9a64-d0866ce123d4)

Did you face the same issue? How to solve it?

**Comment by Mahnaz-Meem** [comment]
This is my github link: https://github.com/Mahnaz-Meem/geohub
But I think it is not relevant in this case.

**Comment by giswqs** [comment]
Try 'pip install -U localtileserver'

**Reply by Mahnaz-Meem** [reply]
This did not work! :(
